### ISSUE TITLE
Enumerations cannot be used in styleF domains

### ISSUE THREAD

**Issue by ochrons** [japgolly/scalacss#101]
Due to the requirement of Universal Equality, enumerations cannot be used for domains in `styleF`. The documentation is very thin on this issue and doesn't provide enough information on how to get Enumeration working with `styleF`.

**Comment by japgolly** [comment]
As in a Scala stdlib Enumeration? Does universal equality always hold (does
== always work meaningfully) for stdlib Enumerations, or does it depend on
anything in the definition?

**Comment by ochrons** [comment]
Yeah, in SPA tutorial I'm using Scala Enumeration. Following seems to pass:

```scala
  implicit val styleUnivEq: UnivEq[CommonStyle.Value] = new UnivEq[CommonStyle.Value] {}

  def commonStyle[A: UnivEq](domain: Domain[A], base: String) = styleF(domain)(opt =>
    styleS(addClassNames(base, s"$base-$opt"))
  )
```

where
```scala
  object CommonStyle extends Enumeration {
    val default, primary, success, info, warning, danger = Value
  }
```

**Comment by japgolly** [comment]
Cool, that should Just Work; that's an oversight. I'll update UnivEq this
week to handle that automatically, before I release it for Scala 2.12.

**Comment by japgolly** [comment]
Fixed in UnivEq 1.0.2 & ScalaCSS 0.5.1
